common/lib/xmodule/xmodule/course_module.py

```diff
@@ -679,7 +679,7 @@ class CourseFields(object):
         ),
         scope=Scope.settings,
         # Ensure that courses imported from XML keep their image
-        default="images_course_image.jpg"
+        default=""
```

Comment:
new default is empty string, because there is the default image that shows up if none is set.